build/publish/PublishNupkgToTransportFeed.targets

```diff
@@ -1,28 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildTasksFeedToolTargetsFileName>"$(NuGetPackagesDir)/microsoft.dotnet.build.tasks.feed/$(BuildTasksFeedToolVersion)/build/Microsoft.DotNet.Build.Tasks.Feed.targets"</BuildTasksFeedToolTargetsFileName>
+  </PropertyGroup>
+
+  <Import Condition="Exists('$(BuildTasksFeedToolTargetsFileName)')" Project="$(BuildTasksFeedToolTargetsFileName)" />
+
   <ItemGroup>
     <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.Cli.Utils.*.nupkg" />
     <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.nupkg" />
   </ItemGroup>
 
   <PropertyGroup>
-    <RelativePath>packages</RelativePath>
+    <ExpectedFeedUrl>https://dotnetfeed.blob.core.windows.net/$(TransportFeedContainerName)/index.json</ExpectedFeedUrl>
```

Comment:
Should this be configurable? Like, for MSRCs? I would probably add a condition here to only set this if the property is empty already. This way we could overwrite this in build definitions if necessary.

Reply:
Yes it will need to be configurable.

Reply:
The "TransportFeedContainerName" is configurable; for testing I've replaced it with "jbeisner".
See: https://github.com/johnbeisner/cli/blob/PublishToTFMethodUpdate/build/AzureInfo.props#L15

"TransportFeedContainerName" has a default of "dotnet-core" and only sets this if the property is empty; however if we change "TRANSPORTFEED_STORAGE_CONTAINER" in build launch we can publish to an 'MSRC' container.

Reply:
I believe the entire URL will need to be configurable.

Reply:
I've made the "ExpectedFeedUrl" fully configurable.